app/main.py

```diff
@@ -1,13 +1,119 @@
+                "column": 80,
+                "message": "line too long (99 > 79 characters)",
+                "name": "E501",
+                "source": "flake8",
+            },
+            {
+                "line": 18,
+                "column": 100,
+                "message": "no newline at end of file",
+                "name": "W292",
+                "source": "flake8",
+            },
+        ],
+        "path": "./source_code_2.py",
+        "status": "failed",
+    },
+    {
+        "errors": [
+            {
+                "line": 3,
+                "column": 74,
+                "message": "multiple statements on one line (semicolon)",
+                "name": "E702",
+                "source": "flake8",
+            },
+            {
+                "line": 3,
+                "column": 80,
+                "message": "line too long (97 > 79 characters)",
+                "name": "E501",
+                "source": "flake8",
+            },
+            {
+                "line": 15,
+                "column": 1,
+                "message": "expected 2 blank lines, found 1",
+                "name": "E302",
+                "source": "flake8",
+            },
+            {
+                "line": 27,
+                "column": 1,
+                "message": "too many blank lines (6)",
+                "name": "E303",
+                "source": "flake8",
+            },
+            {
+                "line": 31,
+                "column": 80,
+                "message": "line too long (99 > 79 characters)",
+                "name": "E501",
+                "source": "flake8",
+            },
+        ],
+        "path": "./source_code_1.py",
+        "status": "failed",
+    },
+    {
+        "errors": [
+            {
+                "line": 4,
+                "column": 1,
+                "message": "expected 2 blank lines, found 1",
+                "name": "E302",
+                "source": "flake8",
+            },
+            {
+                "line": 32,
+                "column": 80,
+                "message": "line too long (84 > 79 characters)",
+                "name": "E501",
+                "source": "flake8",
+            },
+            {
+                "line": 112,
+                "column": 6,
+                "message": "no newline at end of file",
+                "name": "W292",
+                "source": "flake8",
+            },
+        ],
+        "path": "./test_source_code_1.py",
+        "status": "failed",
+    },
+]
 
 
-def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+def format_linter_error(error: Dict[str, Any]) -> Dict[str, Any]:
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
+
+
+def format_single_linter_file(file_path: str, errors: List[Dict[str, Any]]) \
+        -> Dict[str, Any]:
```

Comment:
Do not use `\` for line breaking here and in the `format_linter_report` function as well. Place each argument on a new line instead